app/javascript/components/Courses/CoursesIndex.jsx

```diff
@@ -0,0 +1,137 @@
+
+	 renderCourseUrl = (cell, row) => <a href={row.path}>{cell}</a>;
+
+	 renderCourseOrgLink = (cell, row) => {
+		const url = `https://github.com/${cell}`;
+		return (
+			<a href={url}>{cell}</a>
+		)
+	};
+
+	 renderEditButton = (cell, row) => {
+		return (
+			<Button className="btn btn-warning" variant="warning" data-testid={`edit-button-${row.id}`} href={`${row.edit_path}`} >Edit</Button>
+		)
+	}
+
+	 renderDeleteButton = (cell, row) => {
+		return (
+			<Button className="btn btn-danger" variant="danger" data-testid={`delete-button-${row.id}`} onClick={() => {if(window.confirm(`Delete course ${row.name}?`)) {this.deleteCourse(row)}}} >Delete</Button>
+		)
+	}
+
+	 columns = [
+		{
+			dataField: 'school.abbreviation',
+			text: 'School',
+			sort: true
+		}, {
+			dataField: 'name',
+			text: 'Name',
+			sort: true,
+			formatter: (cell, row) => this.renderCourseUrl(cell, row)
+		}, {
+			dataField: 'term',
+			text: 'Term',
+			sort: true
+		}, {
+			dataField: 'hidden',
+			text: 'Hidden',
+			sort: true
+		}, {
+			dataField: 'course_organization',
+			text: 'Course Organization',
+			sort: true,
+			formatter: (cell, row) => this.renderCourseOrgLink(cell, row)
+		}, {
+			dataField: "edit",
+			text: "Edit",
+			isDummyField: true,
+			formatter: (cell, row) => this.renderEditButton(cell, row),
+			hidden: false /* should be true if this is not an instructor / admin */
+		}, {
+			dataField: "delete",
+			text: "Delete",
+			isDummyField: true,
+			formatter: (cell, row) => this.renderDeleteButton(cell, row),
+			hidden: false /* should be true if this is not an instructor / admin */
+		}
+	];
+
+
+	componentDidMount() {
+		this.getCourses();
+	}
+
+	getCourses = () => {
+		axios.get(coursesRoute).then(response => {
+			this.setState({ courses: response.data });
+		});
+	}
+
+	deleteCourse = course => {
+		axios.delete(course.path).then(_ => this.getCourses());
+	}
+
+	render() {
+		return (
+
+			<BootstrapTable
+				boostrap4={true}
+				columns={this.columns}
+				data={this.state.courses}
+				keyField="id"
+				id="course_list"
+			/>
+
+			// <Fragment>
+			// 	<div>
+			// 		<div className="panel panel-body">
+			// 			<table className="bootstrap-table" data-toggle="table" data-search="true" data-pagination="true" data-show-columns="true" data-show-columns-toggle-all="true" data-show-pagination-switch="true" data-filter-control="true">
+			// 				<thead>
+			// 					<tr>
+			// 						<th scope="col">School</th>
+			// 						<th scope="col">Name</th>
+			// 						<th scope="col">Term</th>
+			// 						<th scope="col">Hidden</th>
+			// 						<th scope="col">GitHub Organization</th>
+			// 						<th scope="col" colSpan="2">Actions</th>
+			// 					</tr>
+			// 				</thead>
+			// 				<tbody>
+			// 					{this.state.courses.map(course =>
+			// 						<CoursesIndexItem course={course} onDelete={this.deleteCourse} key={course.id} />
+			// 					)}
+			// 				</tbody>
```

Comment:
Varun:

Please consider removing this commented out code before we merge this into the master branch

Reply:
I believe this change has been pushed by you, thank you for doing so. My apologies for not catching it.